pkg/cni/cni_iface.go

```diff
@@ -86,56 +81,3 @@ func discover(clusterCIDR string) (*Interface, error) {
 
 	return nil, errors.Errorf("unable to find CNI Interface on the host which has IP from %q", clusterCIDR)
 }
-
-func AnnotateNodeWithCNIInterfaceIP(nodeName string, clientSet kubernetes.Interface, clusterCidr []string) error {
```

Comment:
I assume that after upgrade we'll have orphaned/useless cni IP address annotation that won't be deleted , 
do we want to handle it ?

Reply:
Yep, adding release note sounds reasonable.